src/System.Management.Automation/engine/PSVersionInfo.cs

```diff
@@ -57,6 +57,11 @@ static PSVersionInfo()
         {
             s_psVersionTable = new PSVersionHashTable(StringComparer.OrdinalIgnoreCase);
 
+            string assemblyPath = typeof(PSVersionInfo).GetTypeInfo().Assembly.Location;
```

Comment:
I think `GetTypeInfo()` is not necessary now.
Another way to get the information version string is:
```CSharp
var infoVersion = typeof(PSVersionInfo).Assembly.GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), inherit: false)
infoVersion.InformationalVersion
```

Reply:
`GetTypeInfo()` can be removed.